app/models/cti/caller_id.rb

```diff
@@ -23,10 +23,9 @@ def self.maybe_add(data)
         o_id:      data[:o_id],
         user_id:   data[:user_id],
       )
-      if record.new_record?
-        record.comment = data[:comment]
-        record.save!
-      end
+      return if !record.new_record?
+      record.comment = data[:comment]
+      record.save!
```

Comment:
I prefer the previous version that always returns the record. If you want to keep you version better write `return record record if record.persisted?` or `return record unless record.new_record?`